src/plugins/data/common/search/tabify/response_writer.ts

```diff
@@ -65,9 +65,37 @@ export class TabbedAggResponseWriter {
     }
   }
 
-  response(): TabbedTable {
+  response(): Datatable {
```

Comment:
Maybe worth adding a couple of unit tests for the new implementation of this `response()` method.